catkin_ws/src/motor_control/Motor_PWM.py

```diff
@@ -0,0 +1,10 @@
+import numpy as np
+
+data = np.genfromtxt("Motor PWM Width - Thrust Relation.csv", delimiter=",")
+#microseconds
+pwmSignals = data[:, 0]
+#pounds
+motorThrusts = data[:, 1]
+
+def getPWMSignalWidth(thrust):
```

Comment:
We should make this somewhat explicit with what the units are, since both lbs and kg are in the file. Something that someone can see from the function call itself.